gr-digital/python/digital/gfsk.py

```diff
@@ -80,9 +83,12 @@ def __init__(self,
         ntaps = 4 * samples_per_symbol			# up to 3 bits in filter at once
         #sensitivity = (pi / 2) / samples_per_symbol	# phase change per bit = pi / 2
 
+        
+        if to_unpack:
+            self.unpack = blocks.packed_to_unpacked_bb(1, gr.GR_MSB_FIRST)
```

Comment:
This can move down to the other `if` statement with the `connect()`.

Reply:
Done as recommended